packages/fxa-auth-server/fxa-oauth-server/lib/keys.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const crypto = require('crypto');
+const generateRSAKeypair = require('keypair');
+const JwTool = require('fxa-jwtool');
+
+module.exports = {
+  generatePrivateKey() {
+    const kp = generateRSAKeypair();
+    // We tag our keys with their creation time, and a unique key id
+    // based on a hash of the public key and the timestamp.  The result
+    // comes out like:
+    //  {
+    //    kid: "2017-03-16-ebe69008de771d62cd1cadf9faa6daae"
+    //    "fxa-createdAt": 1489716000,
+    //  }
+    const now = new Date();
+    const pubKeyHash = crypto
+      .createHash('sha256')
+      .update(kp.public)
+      .digest('hex')
+      .slice(0, 32);
+    const privKey = JwTool.JWK.fromPEM(kp.private, {
```

Comment:
If you want to stop using fxa-jwtool, you can use `pem2jwk` from the `pem-jwk` module.